lms/envs/devstack.py

```diff
@@ -215,6 +215,8 @@ def should_show_debug_toolbar(_):
 CORS_ORIGIN_WHITELIST = ()
 CORS_ORIGIN_ALLOW_ALL = True
 
+################################ CCX ###############################
+FEATURES['CUSTOM_COURSES_EDX'] = True
```

Comment:
I was planning to enable this on devstack. It's live in both our prod envs -- I figured there was no reason to keep it off of devstack.

Should I remove it here and make a separate PR to master for it?